docs/src/overview/structure.md

```diff
@@ -50,3 +50,9 @@ There always must be one Application container in a cluster, but there may be mo
 Each Application container corresponds to a `.platform.app.yaml` file in the repository.  If there are 3 `.platform.app.yaml` files, there will be three Application containers.  Application containers hold the code you provide via your Git repository.  Application containers are always built off of one of the Platform.sh-provided language-specific images, such as “PHP 5.6”, “PHP 7.2”, or “Python 3.7”. It is also possible to have multiple Application containers running different languages or versions.
 
 For typical applications, there is only one `.platform.app.yaml` file, which is generally placed at the repository root.
+
+## Dedication
+
+The Dedication is gorgeous if you want a mature and extensive application and scalable solution in the cloud.
```

Comment:
"Dedicated Production Cluster".

Also, "gorgeous" is a bit too markety for the documentation. :-)

Reply:
:) Thank you